data/eos-companion-app.service

```diff
@@ -6,7 +6,29 @@ ConditionPathExists=/etc/avahi/services/companion-app.service
 [Service]
 ExecStart=/usr/bin/dbus-run-session /usr/bin/flatpak run --no-desktop --no-a11y-bus --no-documents-portal com.endlessm.CompanionAppService
 User=companion-app-helper
+# This does not appear to cause any problems when writing
+# to /var/lib/eos-companion-app, remove if it does. We need
+# CAP_SETUID and CAP_SETGID as bwrap is setuid
+CapabilityBoundingSet=CAP_NET_ADMIN CAP_SYS_ADMIN CAP_SYS_CHROOT CAP_SETUID CAP_SETGID
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateDevices=yes
+# Breaks xapian-bridge
+#PrivateNetwork=yes
```

Comment:
Nitpick: Maybe flip the sense of this, and instead do:
```
# xapian-bridge needs local HTTP networking
PrivateNetwork=no
```

Reply:
Good idea (also good to add more of an explanation in the comment)